assets/javascripts/modules/track/trackDeliveryFields.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
It's best to be consistent with using semicolons in ES6 derivative modules, either use them on every line or only when inlining.